kvmagent/kvmagent/plugins/shared_block_plugin.py

```
@@ -1785,4 +1826,105 @@ def _check(cur_vg):
+    def prefix_rebase_backing_files(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = PrefixRebaseBackingFilesRsp()
+
+        old_prefix = cmd.oldPrefix
+        new_prefix = cmd.newPrefix
+
+        if not old_prefix or not new_prefix:
+            raise Exception("oldPrefix and newPrefix must not be empty")
+
+        # Phase 1: Discovery — 收集所有需要 rebase 的 (current, new_backing) 对
+        # 同时收集所有需要激活的 LV 路径（含 image cache）
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)